### PR TITLE
Add plm iorder flag back to PLM

### PR DESCRIPTION
**Pull request by marchdf**
Also snuck in a readme update

### CODE REVIEW

**Review by jrood-nrel — APPROVED** [review]

File: Source/PeleC.cpp [hunk 1]
```diff
@@ -336,6 +336,13 @@ PeleC::read_params()
     amrex::Error("PeleC::ppm_type must be 1 (PPM) to use WENO method");
   }
 
+  if (do_hydro) {
+    if (do_mol || ppm_type == 0) {
+      if ((plm_iorder != 1) && (plm_iorder != 2) && (plm_iorder != 4))
+        amrex::Error("PeleC::plm_iorder must be 1, 2, or 4");
```
Comment (baperry2) [inline Source/PeleC.cpp]: I think for MOL we only really support 1 or 2, so there should be separate checks here for MOL and PLM

**Review by baperry2 — CHANGES_REQUESTED** [review]

File: Source/PeleC.cpp (hunk 1 above)
Comment (marchdf) [reply Source/PeleC.cpp]: good idea

File: Source/PeleC.cpp [hunk 2]
```diff
@@ -337,9 +337,14 @@ PeleC::read_params()
...
 
   if (do_hydro) {
-    if (do_mol || ppm_type == 0) {
-      if ((plm_iorder != 1) && (plm_iorder != 2) && (plm_iorder != 4))
-        amrex::Error("PeleC::plm_iorder must be 1, 2, or 4");
+    if (ppm_type == 0) {
+      if ((plm_iorder != 1) && (plm_iorder != 2) && (plm_iorder != 4)) {
+        amrex::Error("PeleC::plm_iorder for ppm_type = 0 must be 1, 2, or 4");
+      }
+    } else if (do_mol) {
```
Comment (baperry2) [inline Source/PeleC.cpp]: Shouldn't `do_mol` be the `if` and `ppm_type` be the `else if` here?

Comment (marchdf) [reply Source/PeleC.cpp]: nice. good point. I think I fixed it?

**Review by baperry2 — APPROVED** [review]

**Pull request merged**